TrackPropagation/RungeKutta/interface/defaultRKPropagator.h

```diff
@@ -43,7 +43,7 @@ namespace defaultRKPropagator {
   };
   
   struct Product {
-    explicit Product(const MagneticField* field,PropagationDirection dir = alongMomentum, double tolerance = 5.e-5) : 
+    explicit Product(const MagneticField* field,PropagationDirection dir = alongMomentum, double tolerance = 5.e-8) : 
```

Comment:
I guess that this is the origin of regressions in almost all of the tested workflows.
What was the reason to make this change now?

Reply:
ooppps, was an unrelated test in the same directory....